composer.json

```diff
@@ -78,14 +79,12 @@
         "sensio/distribution-bundle": "~5.0.6"
     },
     "require-dev": {
-        "doctrine/doctrine-fixtures-bundle": "~2.3",
         "jackalope/jackalope-doctrine-dbal": "^1.2.5",
-        "massive/build-bundle": "~0.3.0",
         "matthiasnoback/symfony-config-test": "^1.1",
         "matthiasnoback/symfony-dependency-injection-test": "^0.7.3",
         "phpcr/phpcr-shell": "~1.0",
         "phpdocumentor/reflection-docblock": "~3.1",
-        "phpspec/prophecy": "^1.4",
+        "phpspec/prophecy": "^1.7",
```

Comment:
Error in: https://travis-ci.org/sulu/sulu/jobs/355281516#L872. For php 7.1 it need atleast prophecy 1.7 (https://github.com/phpspec/prophecy/issues/314)